netpyne_ui/netpyne_geppetto.py

```diff
@@ -566,6 +571,66 @@ def importModel(self, modelParameters):
         finally:
             os.chdir(owd)
 
+    def importNeuroML(self, modelParameters):
```

Comment:
Those two functions look kinda similar, would it make sense to refactor the code with a high-order function taking `*args` and `**kwargs` and passing them to the function passed as parameter?
That would induce the creation of a function with this signature `def _convert(self, f, *args, **kwargs)` and changing the code of `importNeuroML` and `importLEMS` to call `_convert`. Perhaps it's not necessary here or I missed other variations in the code, or it would introduce an unecessary refactoring that could introduce more complication to fix later bugs?

Reply:
agree, I would open a card for technical debt for this since I don't want to hold the release for this if possible. Thanks

Reply:
the reason for this is the code for importNeuroML and importLEMS was initially submitted and it's currently adapted from Padraig's functions so it's easier to keep the correspondence. BTW agree longer term it would be nice to keep consistency